packages/dds/tree/src/feature-libraries/forest-summary/incrementalSummaryBuilder.ts

```diff
@@ -396,76 +396,75 @@ export class ForestIncrementalSummaryBuilder implements IncrementalEncoderDecode
 			return [];
 		}
```

Comment:
Is this special case necessary? It will do the same thing if we just delete this code right?

Reply:
Yes, you are right. This is not needed anymore.